requirements/env_climada.yml

```diff
@@ -31,7 +31,6 @@ dependencies:
   - salib>=1.3.0
   - scikit-learn>=1.0
   - scipy>=1.6
-  - sparse
```

Comment:
From my experience, it is better to have `conda` handle as many packages as possible, because it tries to create an environment that is compatible to all packages. It seems to make a better job than `pip`. Also, where does the version restriction come from? I did not check explicitly but from the [changelog](https://sparse.pydata.org/en/stable/changelog.html) I do not really see a reason why lower versions should not be supported

Reply:
As for letting `conda` handle it: agreed! (I thought it had eventually been installed from the pypi channel even when listed above, but that was a mistake.)

As for the version restrictions: they are always made quite blindly without further thought. The reason for them is that `conda` i.g. cannot cope well with large possibility spaces and tends to get lost there forever or runs out of memory when versions are defined too liberally. (The situation is better with `mamba` though but that's not the officially suggested environment tool.)

Reply:
I see, that makes sense! Thanks a lot for the review! Then I'll go ahead and merge 😇